css/style.css

```diff
@@ -88,6 +88,16 @@ ul.stacked-list {
   padding: 1rem;
 }
 
+.rating-container {
+  width: 14rem;
+  height: 4rem;
+  /* background: var(--gray); */
```

Comment:
you can remove the commented properties!!

Reply:
Oh! yes 👍